test/example-types/metricGroups/pivot.test.js

```diff
@@ -48,66 +89,39 @@ describe('pivot', () => {
-        { type: 'avg', field: 'LineItem.TotalPrice' },
-        { type: 'sum', field: 'LineItem.TotalPrice' },
-        {
-          type: 'percentiles',
-          field: 'LineItem.TotalPrice',
-          percents: [20, 50],
-        },
-        { type: 'cardinality', field: 'Vendor.Name' },
-        { type: 'topHits' },
-      ],
-      rows: [
-        { type: 'fieldValues', field: 'Organization.NameState' },
-        { type: 'dateInterval', field: 'PO.IssuedDate', interval: 'month' },
-      ],
-    }
+    let { buildQuery } = createPivotScope(
+      pivotTestNode({
+        values: [
+          // labels are `${_.startCase(type)} of ${schemaLabel(field)}`
+          // key - defaults to `${type}-${field}`
+          //    unless type has a special override? e.g. if doesnt take field, like top_hits
+          { type: 'min', field: 'LineItem.TotalPrice' },
+          { type: 'max', field: 'LineItem.TotalPrice' },
+          { type: 'avg', field: 'LineItem.TotalPrice' },
+          { type: 'sum', field: 'LineItem.TotalPrice' },
+          {
+            type: 'percentiles',
+            field: 'LineItem.TotalPrice',
+            percents: [20, 50],
+          },
+          { type: 'cardinality', field: 'Vendor.Name' },
+          { type: 'topHits' },
+        ],
+        rows: [
+          { type: 'fieldValues', field: 'Organization.NameState' },
+          { type: 'dateInterval', field: 'PO.IssuedDate', interval: 'month' },
+        ],
+        columns: [],
+      }),
+      testSchemas(['Organization.NameState', 'Vendor.Name']),
+      getStats
+    )
+
     let expected = {
       aggs: {
         rows: {
           terms: { field: 'Organization.NameState.untouched', size: 10 },
           aggs: {
-            rows: {
-              date_histogram: {
-                field: 'PO.IssuedDate',
-                interval: 'month',
-                min_doc_count: 0,
-              },
-              aggs: {
-                'pivotMetric-min-LineItem.TotalPrice': {
-                  min: { field: 'LineItem.TotalPrice' },
-                },
-                'pivotMetric-max-LineItem.TotalPrice': {
-                  max: { field: 'LineItem.TotalPrice' },
-                },
-                'pivotMetric-avg-LineItem.TotalPrice': {
-                  avg: { field: 'LineItem.TotalPrice' },
-                },
-                'pivotMetric-sum-LineItem.TotalPrice': {
-                  sum: { field: 'LineItem.TotalPrice' },
-                },
-                'pivotMetric-percentiles-LineItem.TotalPrice': {
-                  percentiles: {
-                    percents: [20, 50],
-                    field: 'LineItem.TotalPrice',
-                  },
-                },
-                'pivotMetric-cardinality-Vendor.Name': {
-                  cardinality: { field: 'Vendor.Name.untouched' },
-                },
-                'pivotMetric-topHits': { top_hits: {} },
-              },
-            },
```

Comment:
why would all of this go away? We should still be evaluating the nested rows when there's no specific expansion

Reply:
Added new flags and reverted and fixed tests